Models/userModel.js

```diff
@@ -0,0 +1,15 @@
+const mongoose = require('../DataBase/connection');
+const UserSchema = mongoose.Schema({
+   
+    name: String,
+    zip: Number,
+    pet: [
+        {
+            petName: String,
+            age: Number,
+            breed: String
+        }
+    ]
```

Comment:
I think we probably need a collection for pets, with its own schema and controllers for creating, updating, and deleting pets.

Reply:
I created the Pet model, controller, and router. The pets section of the user schema should be updated to:
```suggestion
    pet: [
        {
            ref: "Pet",
            type: mongoose.Schema.Types.ObjectId
        }
    ]
```